app/channels/room_channel.rb

```diff
@@ -95,7 +92,13 @@ def render_error_json(message)
     end
 
     def exit_room
-      UserRoomLog.where(user: current_user, room: @room, exit_at: nil).update(exit_at: Time.now.utc)
+      ActiveRecord::Base.transaction do
+        updated_log = UserRoomLog.where(user: current_user, room: @room, exit_at: nil).update(exit_at: Time.now.utc)
+        if updated_log.present? then
```

Comment:
__[RuboCop]__

Style/MultilineIfThen: Do not use `then` for multi-line `if`.


<!-- Comment by [SideCI](https://sideci.com) -->